### PR TITLE
Updates and edits for May 2019 workshop

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Suggested edits to clarify and correct bits of this workshop - please check! Thanks.

### CODE REVIEW

File: 01-getting-started/README.md [hunk 1]
```diff
@@ -51,8 +65,8 @@ You can verify your setup by running the version command.
 
 ```console
 $ helm version
-Client: &version.Version{SemVer:"v2.13.0", GitCommit:"79d07943b03aea2b76c12644b4b54733bc5958d6", GitTreeState:"clean"}
-Server: &version.Version{SemVer:"v2.13.0", GitCommit:"79d07943b03aea2b76c12644b4b54733bc5958d6", GitTreeState:"clean"}
+Client: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
```
Comment (thomastaylor312) [inline 01-getting-started/README.md:68]: Ha! I forgot that happened after I pushed

File: 02-installing-charts/README.md [hunk 2]
```diff
@@ -260,7 +265,7 @@ There is also a shortcut that we will use here to rollback to the last release:
 
 ```console
 $ helm rollback <release_name> 0
-Rollback was a success! Happy Helming!
```
Comment (thomastaylor312) [inline 02-installing-charts/README.md:263]: Nit: This is the actual output from Helm, so I probably want to keep it.

**Review by thomastaylor312 — APPROVED** [review]
These are great changes. Thank you!

File: 02-installing-charts/README.md (hunk 2 above)
Comment (bridgetkromhout) [reply 02-installing-charts/README.md:263]: Weirdly, I was also going with "changing it to the actual output I see" and I think I have the most recent everything. Do you wanna re-check with the versions currently in play? I wonder where the disconnect is. (I'm on a Mac if that helps.)

Comment (thomastaylor312) [reply 02-installing-charts/README.md:263]: I think it is a change in 2.14 then! Good catch

**Pull request merged**